src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/ObjectToIdCache.cs

```diff
@@ -171,6 +150,9 @@ private static bool IsPrime(int candidate)
         {
             3, 7, 17, 37, 89, 197, 431, 919, 1931, 4049, 8419, 17519, 36353,
             75431, 156437, 324449, 672827, 1395263, 2893249, 5999471,
+            11998949, 23997907, 47995853, 95991737, 191983481, 383966977, 767933981, 1535867969,
+            2146435069, 0X7FEFFFFF
```

Comment:
@JonHanna Why these two numbers are after 1535867969? Isn't it only 2147483423, which is the largest prime that is less than Int32.MaxValue?